docs/keys_and_certificates.md

```diff
@@ -221,15 +220,40 @@ keyUsage               = critical, keyCertSign, cRLSign
 subjectKeyIdentifier   = hash
 authorityKeyIdentifier = keyid:always
 ```
+Feel free to adjust configuration to your needs.
+
+* Run command:
+    ```shell
+    openssl req -newkey rsa:2048 -nodes -keyout root_ca_priv.key -x509 -days 365 \
+    -out root_ca.crt -config root_ca.cfg
+    ```
+    - `root_ca_priv.key` - newly created private root CA key. Keep it safe.
+    - `root_ca.crt` - root CA certificate. This will be hardcoded and marked as
+      trusted by Fobnail Token.
+
 
-### Intermediate/issuing CA
+### Intermediate/PO issuing CA
```

Comment:
I like how the Root CA section was improved. It would be great to have the same format for Intermediate/PO issuing CA.

Reply:
It may be hard because one of them is optional and depending on whether it is or isn't used, instructions for the other one will be different. I'll see what I can do.

Reply:
Sane copypastable example and existing explanation as part of how to, if one would like something more customized. TL;DR fix the problem, but this section is off in comparison to Root CA.

Reply:
Changed in https://github.com/fobnail/docs/pull/47/commits/a9003b6b8e3577462bdd9a13e17b895724ea6930